programs/programs/urgent_needs/il/il_beacon.py

```diff
@@ -0,0 +1,22 @@
+from ..base import UrgentNeedFunction
+
+
+class il_beacon(UrgentNeedFunction):
+    dependencies = ["age"]
+    min_age = 5
+    max_age = 21
+
+    def eligible(self):
+        # Check if household has marked mental health or child development help
+        needs_mental_health = self.screen.needs_mental_health_help
+        needs_child_dev = self.screen.needs_child_dev_help
+
+        # Check if at least one household member is between 5-21
+        has_eligible_age_member = False
+        for member in self.screen.household_members.all():
```

Comment:
Rather than looping through the household members I think you can use this helper:
`self.screen.num_children(age_min=self.min_age, age_max=self.max_age) > 0`